md/howto-debugcustompage.md

```diff
@@ -0,0 +1,89 @@
+# How to Debug a custom page ? 
+
+<h2></h2>
+
+##  Question
```

Comment:
You need to write an overview sentence to explain the context and the added-value of the page. It is not obvious for everybody.